### DISCUSSION TITLE
v3.4 | WE'RE BACK BABY

### DISCUSSION THREAD

**Discussion by xEdziu** [xEdziu/KeepMyPassword-Desktop#162] [Announcements]
# HI EVERYONE - GOOD TO SEE YOU AGAIN

After some time I finnally managed to get myself together and repair the things that were not working properly. Now y'all should be able to use it without any issues. I am sorry that you waited so long :(

Thanks @Kubis10 for forcing me to sit down and do that once and for good.

## What's Changed
* Upgrade CodeSee workflow to version 2 by @codesee-maps in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/152
* Install the CodeSee workflow. Learn more at https://docs.codesee.io (… by @xEdziu in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/153
* Greek translation and fixed 'cannot register' for Windows by @alexia-nt in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/155
* Bump sqlite-jdbc from 3.36.0.3 to 3.41.2.2 by @dependabot in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/161
* Avoid Duplication Of Methods by @hardaman in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/160
* Repaired error while registering on some usernames

## New Contributors
* @codesee-maps made their first contribution in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/152
* @alexia-nt made their first contribution in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/155
* @dependabot made their first contribution in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/161
* @hardaman made their first contribution in https://github.com/xEdziu/KeepMyPassword-Desktop/pull/160

**Full Changelog**: https://github.com/xEdziu/KeepMyPassword-Desktop/compare/v3.3.2...v3.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/xEdziu/KeepMyPassword-Desktop/releases/tag/v3.4.0'>v3.4 | WE'RE BACK BABY</a>.</em>

**Comment by Kubis10** [comment]
HYPE

**Comment by xEdziu** [comment]
So far it's prelease because I am waiting for compiled macOS version, but as soon as it will be done - WE ARE GOING FOR IT
